### PR TITLE
Set up new rails project

### PR DESCRIPTION
**Pull request by koggs**
#### What I did:
- Create new project
- Using Postrgres database
- Add rubocop

### CODE REVIEW

File: eventplanner/Gemfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Use postgresql as the database for Active Record
+gem 'pg', '>= 0.18', '< 2.0'
+# Use Puma as the app server
+gem 'puma', '~> 3.0'
+# Use SCSS for stylesheets
+gem 'sass-rails', '~> 5.0'
+# Use Uglifier as compressor for JavaScript assets
+gem 'uglifier', '>= 1.3.0'
+# Use CoffeeScript for .coffee assets and views
+gem 'coffee-rails', '~> 4.2'
```
Comment (juliaramstedt) [inline eventplanner/Gemfile]: you can remove this if you don't plan on writing coffee script

File: eventplanner/Gemfile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+# See https://github.com/rails/execjs#readme for more supported runtimes
+# gem 'therubyracer', platforms: :ruby
+
+# Use jquery as the JavaScript library
+gem 'jquery-rails'
```
Comment (juliaramstedt) [inline eventplanner/Gemfile]: you can remove this if you don't plan on writing jquery

File: eventplanner/Gemfile (hunk 1 above)
Comment (koggs) [reply eventplanner/Gemfile]: I had a feeling I was forgetting something :)

File: eventplanner/Gemfile (hunk 2 above)
Comment (koggs) [reply eventplanner/Gemfile]: 👍

**Pull request merged**